assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/usePrice.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useMemo } from 'react';
+
+import { useDataState } from '../data';
+import { BaseFieldProps, PriceFieldProps, UsePrice } from './types';
+
+type BFP = BaseFieldProps;
+
+const usePrice = ({ field, price }: PriceFieldProps): UsePrice => {
+	const { updatePrice } = useDataState();
+
+	const getValue: BFP['getValue'] = useCallback(() => price[field], [price[field]]);
+
+	const setValue: BFP['setValue'] = useCallback(
```

Comment:
To set better type safety, it's good to use `useCallback` generic to set types for internal callback and the variable will also infer the type from it.
```suggestion
	const getValue = useCallback<BFP['getValue']>(() => price[field], [price[field]]);

	const setValue = useCallback<BFP['setValue']>(
```